internal/config/config_type.go

```diff
@@ -167,6 +177,35 @@ func (c *fileConfig) Write() error {
 	return WriteConfigFile(ConfigFile(), marshalled)
 }
 
+func (c *fileConfig) Aliases() (*AliasConfig, error) {
+	_, aliasesEntry, err := c.FindEntry("aliases")
+	var nfe *NotFoundError
+	if err != nil && errors.As(err, &nfe) {
+		// TODO does this make sense at all? want to simulate existing but empty key.
+		return &AliasConfig{Parent: c}, nil
+	}
+
+	aliasConfig, err := c.parseAliasConfig(aliasesEntry)
+	if err != nil {
+		return nil, err
+	}
+
+	return aliasConfig, nil
+}
+
+func (c *fileConfig) parseAliasConfig(aliasesEntry *yaml.Node) (*AliasConfig, error) {
+	if aliasesEntry.Kind != yaml.MappingNode {
+		return nil, errors.New("expected aliases to be a map")
```

Comment:
Could we also interpret a YAML null essentially as an empty mapping?

When experimenting with aliases, I've manually deleted alias entries in my config file, resulting in only this:
```yaml
aliases:
```
the current alias-parsing code now aborts because `aliases` is not a map, but I think we should be permissive about this and interpret null as an empty map.

Reply:
oops yeah, good catch. that shouldn't explode a config parsing.